app/views/unattended/provisioning_templates/host_init_config/host_init_config_default.erb

```diff
@@ -47,7 +47,7 @@ trap 'exit_and_cancel_build' ERR
 
 <% end -%>
 
-<%= install_packages(host_param('host_packages')) -%>
+<%= install_packages(shell_escape(host_param('host_packages'))) -%>
```

Comment:
Looking at the definition, it expects a space delimited list of package names. Won't this break if we escape the it?